starter-code/iteration2/src/app/app.module.ts

```diff
@@ -0,0 +1,22 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+
+
+import { AppComponent } from './app.component';
+import { MainComponentComponent } from './main-component/main-component.component';
```

Comment:
no need of naming a component component, as it will be added by angular cli automatic as you can see.